raft/raft.go

```diff
@@ -166,10 +190,22 @@ func (raft *Raft) candidateSelect() {
 			///////////////////
 			//  MODIFY HERE  //
 
+			if(rvr.Term>raft.currentTerm){
```

Comment:
http://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak
"a small change that may not be very important, but is technically correct" ahhhh